doc/.custom_wordlist.txt

```diff
@@ -163,6 +170,7 @@ OverlayFS
 OVMF
 OVN
 OVS
+passthrough
```

Comment:
Use "pass-through" instead (this doesn't need to be added to the wordlist).

Reply:
fixed